product/le.dyn.js

```diff
@@ -125,7 +125,7 @@ var XMLHttpRequest = function() {
         /**
          * @const
          * @type {string} */
-        var _endpoint = "js.logentries.com/v1";
+        var _endpoint = "js.logs.insight.rapid7.com/v1";
```

Comment:
Did you verify that this endpoint exists and works? Because I'm not entirely sure it does...

Reply:
It comes from the code found in `product/le.js` 
https://github.com/rapid7/r7insight_js/blob/0b3a79fca9dc402767bb6be2ca34b1f16cc05051/product/le.js#L89

Reply:
... and you've tested that this works? :P Also how does it know which region to send log files to?

Reply:
Not quite sure why, but `_endpoint` gets another value on the next line 
![image](https://user-images.githubusercontent.com/52921864/69718646-ce7d1000-1106-11ea-9439-828fdfedb9f6.png)